qp_qiime2/__init__.py

```diff
@@ -21,176 +25,171 @@
 }
 qiime_cmd = QiitaCommand(
-    "Rarefy", "Rarefy",
+    "Rarefy features", "Rarefy",
     rarefy, req_params, opt_params, outputs, dflt_param_set,
     analysis_only=True)
 plugin.register_command(qiime_cmd)
 
 # Define the beta_diversity command
-req_params = {'i-table': ('artifact', ['BIOM'])}
+req_params = {'BIOM table': ('artifact', ['BIOM'])}
 opt_params = {
-    'p-metric': [
-        ('choice:["sokalsneath", "kulsinski", "mahalanobis", "matching", '
-         '"euclidean", "correlation", "yule", "russellrao", "hamming", '
-         '"jaccard", "braycurtis", "dice", "rogerstanimoto", "sqeuclidean", '
-         '"cityblock", "sokalmichener", "cosine", "wminkowski", "seuclidean", '
-         '"chebyshev", "canberra", "unweighted UniFrac", '
-         '"weighted normalized UniFrac", "weighted unnormalized UniFrac"]'),
-        'jaccard'],
-    'i-tree': ['choice:["default", "None"]', 'None']}
+    'Diversity metric': [
+        'choice:%s' % dumps(list(BETA_DIVERSITY_METRICS.keys())),
+        'Jaccard similarity index'],
+    'Phylogenetic tree': ['choice:["default", "None"]', 'None']}
 outputs = {'distance_matrix': 'distance_matrix'}
 dflt_param_set = {
     'Defaults': {
-        'p-metric': 'jaccard',
-        'i-tree': 'None'}
+        'Diversity metric': 'Jaccard similarity index',
+        'Phylogenetic tree': 'None'}
 }
 qiime_cmd = QiitaCommand(
-    "beta_diversity", "Beta Diversity",
+    "Calculate beta diversity", "Beta Diversity",
     beta_diversity, req_params, opt_params, outputs, dflt_param_set,
     analysis_only=True)
 plugin.register_command(qiime_cmd)
 
 # Define the pcoa command
-req_params = {'i-distance-matrix': ('artifact', ['distance_matrix'])}
+req_params = {'Distance matrix': ('artifact', ['distance_matrix'])}
 opt_params = {}
 outputs = {'o-pcoa': 'ordination_results'}
 dflt_param_set = {
     'Defaults': {}
 }
 qiime_cmd = QiitaCommand(
-    "pcoa", "Principal Coordinate Analysis",
+    "Generate principal coordinates analysis (PCoA)",
+    "Principal Coordinate Analysis",
     pcoa, req_params, opt_params, outputs, dflt_param_set,
     analysis_only=True)
 plugin.register_command(qiime_cmd)
 
 # Define the beta_correlation command
-req_params = {'i-distance-matrix': ('artifact', ['distance_matrix']),
-              'm-metadata-category': ('string', '')}
-opt_params = {'p-method': ['choice:["spearman", "pearson"]', 'spearman'],
-              'p-permutations': ('integer', 999)}
+req_params = {'Distance matrix': ('artifact', ['distance_matrix']),
+              'Metadata category': ('string', '')}
+opt_params = {'Correlation method':
+              ['choice:%s' % dumps(list(BETA_CORRELATION_METHODS)),
+               'Spearman'],
+              'Number of permutations': ('integer', 999)}
 outputs = {'q2_visualization': 'q2_visualization'}
 dflt_param_set = {
     'Defaults': {
-        'p-method': 'spearman',
-        'p-permutations': 999}
+        'Correlation method': 'spearman',
```

Comment:
isn't the [default](https://github.com/biocore/scikit-bio/blob/master/skbio/stats/distance/_mantel.py#L21) normally pearson? it